[2023-09-08 17:31:22,229: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-09-08 17:31:22,250: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-09-08 17:31:22,325: INFO: common: yaml file: params.yaml loaded successfully]
[2023-09-08 17:31:22,327: ERROR: main: yaml file is empty]
Traceback (most recent call last):
  File "d:\NLP\NLP-Summarize-Text\src\textSummarizer\utils\common.py", line 30, in read_yaml
    return ConfigBox(content)
  File "box\box.py", line 259, in box.box.Box.__init__
box.exceptions.BoxValueError: First argument must be mapping or iterable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:/NLP/NLP-Summarize-Text/main.py", line 10, in <module>
    data_ingestion.main()
  File "d:\NLP\NLP-Summarize-Text\src\textSummarizer\pipeline\stage_01_data_ingestion.py", line 11, in main
    config = ConfigurationManager()
  File "d:\NLP\NLP-Summarize-Text\src\textSummarizer\config\configuration.py", line 17, in __init__
    self.params = read_yaml(params_filepath)
  File "D:\AnacondaProjects\envs\Deep_Learning_38\lib\site-packages\ensure\main.py", line 872, in __call__
    return_val = self.f(*args, **kwargs)
  File "d:\NLP\NLP-Summarize-Text\src\textSummarizer\utils\common.py", line 32, in read_yaml
    raise ValueError("yaml file is empty")
ValueError: yaml file is empty
[2023-09-08 17:32:57,171: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-09-08 17:32:57,175: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-09-08 17:32:57,176: INFO: common: yaml file: params.yaml loaded successfully]
[2023-09-08 17:32:57,186: INFO: common: created directory at: artifacts]
[2023-09-08 17:32:57,187: INFO: common: created directory at: artifacts/data_ingestion]
[2023-09-08 17:32:57,188: ERROR: main: expected string or bytes-like object]
Traceback (most recent call last):
  File "d:/NLP/NLP-Summarize-Text/main.py", line 10, in <module>
    data_ingestion.main()
  File "d:\NLP\NLP-Summarize-Text\src\textSummarizer\pipeline\stage_01_data_ingestion.py", line 14, in main
    data_ingestion.download_file()
  File "d:\NLP\NLP-Summarize-Text\src\textSummarizer\components\data_ingestion.py", line 18, in download_file
    filename, headers = request.urlretrieve(
  File "D:\AnacondaProjects\envs\Deep_Learning_38\lib\urllib\request.py", line 245, in urlretrieve
    url_type, path = _splittype(url)
  File "D:\AnacondaProjects\envs\Deep_Learning_38\lib\urllib\parse.py", line 1030, in _splittype
    match = _typeprog.match(url)
TypeError: expected string or bytes-like object
[2023-09-08 17:35:10,106: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-09-08 17:35:10,111: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-09-08 17:35:10,112: INFO: common: yaml file: params.yaml loaded successfully]
[2023-09-08 17:35:10,112: INFO: common: created directory at: artifacts]
[2023-09-08 17:35:10,113: INFO: common: created directory at: artifacts/data_ingestion]
[2023-09-08 17:35:11,488: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 117A:6CFA:ABE11C:CC6459:64FBBDBB
Accept-Ranges: bytes
Date: Sat, 09 Sep 2023 00:35:11 GMT
Via: 1.1 varnish
X-Served-By: cache-bur-kbur8200097-BUR
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1694219711.106095,VS0,VE299
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 0c6cf0a970eded75c53f4ca4cf2b022a49326fb0
Expires: Sat, 09 Sep 2023 00:40:11 GMT
Source-Age: 0

]
[2023-09-08 17:35:11,769: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-09-20 15:54:59,536: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-09-20 15:54:59,646: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-09-20 15:54:59,667: INFO: common: yaml file: params.yaml loaded successfully]
[2023-09-20 15:54:59,670: INFO: common: created directory at: artifacts]
[2023-09-20 15:54:59,672: INFO: common: created directory at: artifacts/data_ingestion]
[2023-09-20 15:54:59,674: INFO: data_ingestion: File already exists of size: ~ 7718 KB]
[2023-09-20 15:55:00,501: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-09-21 13:08:05,989: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-09-21 13:08:06,101: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-09-21 13:08:06,130: INFO: common: yaml file: params.yaml loaded successfully]
[2023-09-21 13:08:06,141: INFO: common: created directory at: artifacts]
[2023-09-21 13:08:06,143: INFO: common: created directory at: artifacts/data_ingestion]
[2023-09-21 13:08:06,146: INFO: data_ingestion: File already exists of size: ~ 7718 KB]
[2023-09-21 13:08:06,850: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-09-21 13:08:06,852: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2023-09-21 13:08:06,888: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-09-21 13:08:06,893: INFO: common: yaml file: params.yaml loaded successfully]
[2023-09-21 13:08:06,897: INFO: common: created directory at: artifacts]
[2023-09-21 13:08:06,900: INFO: common: created directory at: artifacts/data_validation]
[2023-09-21 13:08:06,908: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2023-09-22 12:06:04,734: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-09-22 12:13:46,896: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-09-22 12:14:09,841: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-09-22 12:14:11,496: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-09-22 12:14:11,517: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-09-22 12:14:11,550: INFO: common: yaml file: params.yaml loaded successfully]
[2023-09-22 12:14:11,552: INFO: common: created directory at: artifacts]
[2023-09-22 12:14:11,552: INFO: common: created directory at: artifacts/data_ingestion]
[2023-09-22 12:14:11,552: INFO: data_ingestion: File already exists of size: ~ 7718 KB]
[2023-09-22 12:14:11,859: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-09-22 12:14:11,859: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2023-09-22 12:14:11,868: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-09-22 12:14:11,870: INFO: common: yaml file: params.yaml loaded successfully]
[2023-09-22 12:14:11,871: INFO: common: created directory at: artifacts]
[2023-09-22 12:14:11,871: INFO: common: created directory at: artifacts/data_validation]
[2023-09-22 12:14:11,873: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2023-09-22 12:14:11,874: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2023-09-22 12:14:11,884: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-09-22 12:14:11,886: INFO: common: yaml file: params.yaml loaded successfully]
[2023-09-22 12:14:11,887: INFO: common: created directory at: artifacts]
[2023-09-22 12:14:11,888: INFO: common: created directory at: artifacts/data_transformation]
[2023-09-22 12:14:14,339: ERROR: main: Couldn't instantiate the backend tokenizer from one of: 
(1) a `tokenizers` library serialization file, 
(2) a slow tokenizer instance to convert or 
(3) an equivalent slow tokenizer class to instantiate and convert. 
You need to have sentencepiece installed to convert a slow tokenizer to a fast one.]
Traceback (most recent call last):
  File "d:/NLP/NLP-Summarize-Text/main.py", line 33, in <module>
    data_transformation.main()
  File "d:\NLP\NLP-Summarize-Text\src\textSummarizer\pipeline\stage_03_data_transformation.py", line 11, in main
    data_transformation = DataTransformation(config=data_transformation_config)
  File "d:\NLP\NLP-Summarize-Text\src\textSummarizer\components\data_transformation.py", line 12, in __init__
    self.tokenizer = AutoTokenizer.from_pretrained(config.tokenizer_name)
  File "D:\AnacondaProjects\envs\Deep_Learning_38\lib\site-packages\transformers\models\auto\tokenization_auto.py", line 637, in from_pretrained
    return tokenizer_class_fast.from_pretrained(pretrained_model_name_or_path, *inputs, **kwargs)
  File "D:\AnacondaProjects\envs\Deep_Learning_38\lib\site-packages\transformers\tokenization_utils_base.py", line 1777, in from_pretrained
    return cls._from_pretrained(
  File "D:\AnacondaProjects\envs\Deep_Learning_38\lib\site-packages\transformers\tokenization_utils_base.py", line 1932, in _from_pretrained
    tokenizer = cls(*init_inputs, **init_kwargs)
  File "D:\AnacondaProjects\envs\Deep_Learning_38\lib\site-packages\transformers\models\pegasus\tokenization_pegasus_fast.py", line 142, in __init__
    super().__init__(
  File "D:\AnacondaProjects\envs\Deep_Learning_38\lib\site-packages\transformers\tokenization_utils_fast.py", line 120, in __init__
    raise ValueError(
ValueError: Couldn't instantiate the backend tokenizer from one of: 
(1) a `tokenizers` library serialization file, 
(2) a slow tokenizer instance to convert or 
(3) an equivalent slow tokenizer class to instantiate and convert. 
You need to have sentencepiece installed to convert a slow tokenizer to a fast one.
[2023-09-22 12:51:01,833: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-09-22 12:51:25,616: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-09-22 12:51:25,654: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-09-22 12:51:25,670: INFO: common: yaml file: params.yaml loaded successfully]
[2023-09-22 12:51:25,672: INFO: common: created directory at: artifacts]
[2023-09-22 12:51:25,674: INFO: common: created directory at: artifacts/data_ingestion]
[2023-09-22 12:51:25,678: INFO: data_ingestion: File already exists of size: ~ 7718 KB]
[2023-09-22 12:51:26,171: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-09-22 12:51:26,172: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2023-09-22 12:51:26,185: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-09-22 12:51:26,186: INFO: common: yaml file: params.yaml loaded successfully]
[2023-09-22 12:51:26,187: INFO: common: created directory at: artifacts]
[2023-09-22 12:51:26,188: INFO: common: created directory at: artifacts/data_validation]
[2023-09-22 12:51:26,192: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2023-09-22 12:51:26,193: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2023-09-22 12:51:26,209: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-09-22 12:51:26,211: INFO: common: yaml file: params.yaml loaded successfully]
[2023-09-22 12:51:26,212: INFO: common: created directory at: artifacts]
[2023-09-22 12:51:26,214: INFO: common: created directory at: artifacts/data_transformation]
[2023-09-22 12:51:41,163: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2023-09-28 12:39:35,881: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-09-28 12:41:00,974: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-09-28 12:48:13,973: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-09-28 12:50:12,910: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-09-28 12:53:10,085: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-09-28 12:53:11,520: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-09-28 12:53:11,572: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-09-28 12:53:11,635: INFO: common: yaml file: params.yaml loaded successfully]
[2023-09-28 12:53:11,643: INFO: common: created directory at: artifacts]
[2023-09-28 12:53:11,644: INFO: common: created directory at: artifacts/data_ingestion]
[2023-09-28 12:53:14,973: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 15B2:0326:5E293D:7205A6:6515D9A5
Accept-Ranges: bytes
Date: Thu, 28 Sep 2023 19:53:14 GMT
Via: 1.1 varnish
X-Served-By: cache-bur-kbur8200054-BUR
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1695930792.380685,VS0,VE1938
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 27faf719ef76677cf566fee55d548d1d184645d8
Expires: Thu, 28 Sep 2023 19:58:14 GMT
Source-Age: 2

]
[2023-09-28 12:53:15,139: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-09-28 12:53:15,140: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2023-09-28 12:53:15,144: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-09-28 12:53:15,147: INFO: common: yaml file: params.yaml loaded successfully]
[2023-09-28 12:53:15,147: INFO: common: created directory at: artifacts]
[2023-09-28 12:53:15,148: INFO: common: created directory at: artifacts/data_validation]
[2023-09-28 12:53:15,150: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2023-09-28 12:53:15,151: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2023-09-28 12:53:15,161: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-09-28 12:53:15,165: INFO: common: yaml file: params.yaml loaded successfully]
[2023-09-28 12:53:15,166: INFO: common: created directory at: artifacts]
[2023-09-28 12:53:15,167: INFO: common: created directory at: artifacts/data_transformation]
[2023-09-28 12:53:22,838: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2023-09-28 12:55:14,218: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-09-28 13:01:54,905: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-09-28 13:02:04,011: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-09-28 13:02:04,036: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-09-28 13:02:04,073: INFO: common: yaml file: params.yaml loaded successfully]
[2023-09-28 13:02:04,083: INFO: common: created directory at: artifacts]
[2023-09-28 13:02:04,085: INFO: common: created directory at: artifacts/data_ingestion]
[2023-09-28 13:02:04,087: INFO: data_ingestion: File already exists of size: ~ 7718 KB]
[2023-09-28 13:02:04,265: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-09-28 13:02:04,266: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2023-09-28 13:02:04,275: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-09-28 13:02:04,278: INFO: common: yaml file: params.yaml loaded successfully]
[2023-09-28 13:02:04,279: INFO: common: created directory at: artifacts]
[2023-09-28 13:02:04,280: INFO: common: created directory at: artifacts/data_validation]
[2023-09-28 13:02:04,282: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2023-09-28 13:02:04,283: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2023-09-28 13:02:04,293: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-09-28 13:02:04,301: INFO: common: yaml file: params.yaml loaded successfully]
[2023-09-28 13:02:04,303: INFO: common: created directory at: artifacts]
[2023-09-28 13:02:04,304: INFO: common: created directory at: artifacts/data_transformation]
[2023-09-28 13:02:07,201: WARNING: arrow_dataset: Loading cached processed dataset at artifacts/data_ingestion/samsum_dataset/train\cache-0cc0f255bad7f608.arrow]
[2023-09-28 13:02:08,211: WARNING: arrow_dataset: Loading cached processed dataset at artifacts/data_ingestion/samsum_dataset/validation\cache-7499891abfa422fb.arrow]
[2023-09-28 13:02:08,335: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2023-09-28 13:06:29,125: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-09-28 13:06:30,736: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-09-28 13:06:30,742: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-09-28 13:06:30,746: INFO: common: yaml file: params.yaml loaded successfully]
[2023-09-28 13:06:30,747: INFO: common: created directory at: artifacts]
[2023-09-28 13:06:30,747: INFO: common: created directory at: artifacts/data_ingestion]
[2023-09-28 13:06:30,748: INFO: data_ingestion: File already exists of size: ~ 7718 KB]
[2023-09-28 13:06:30,877: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-09-28 13:06:30,878: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2023-09-28 13:06:30,893: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-09-28 13:06:30,897: INFO: common: yaml file: params.yaml loaded successfully]
[2023-09-28 13:06:30,897: INFO: common: created directory at: artifacts]
[2023-09-28 13:06:30,898: INFO: common: created directory at: artifacts/data_validation]
[2023-09-28 13:06:30,901: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2023-09-28 13:06:30,902: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2023-09-28 13:06:30,910: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-09-28 13:06:30,914: INFO: common: yaml file: params.yaml loaded successfully]
[2023-09-28 13:06:30,915: INFO: common: created directory at: artifacts]
[2023-09-28 13:06:30,915: INFO: common: created directory at: artifacts/data_transformation]
[2023-09-28 13:06:33,100: WARNING: arrow_dataset: Loading cached processed dataset at artifacts/data_ingestion/samsum_dataset/train\cache-0cc0f255bad7f608.arrow]
[2023-09-28 13:06:33,126: WARNING: arrow_dataset: Loading cached processed dataset at artifacts/data_ingestion/samsum_dataset/test\cache-e51af59ab391f6b0.arrow]
[2023-09-28 13:06:33,488: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2023-09-28 13:07:13,827: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-09-28 13:07:15,397: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-09-28 13:07:15,406: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-09-28 13:07:15,409: INFO: common: yaml file: params.yaml loaded successfully]
[2023-09-28 13:07:15,410: INFO: common: created directory at: artifacts]
[2023-09-28 13:07:15,411: INFO: common: created directory at: artifacts/data_ingestion]
[2023-09-28 13:07:15,411: INFO: data_ingestion: File already exists of size: ~ 7718 KB]
[2023-09-28 13:07:15,535: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-09-28 13:07:15,536: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2023-09-28 13:07:15,544: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-09-28 13:07:15,548: INFO: common: yaml file: params.yaml loaded successfully]
[2023-09-28 13:07:15,549: INFO: common: created directory at: artifacts]
[2023-09-28 13:07:15,550: INFO: common: created directory at: artifacts/data_validation]
[2023-09-28 13:07:15,552: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2023-09-28 13:07:15,554: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2023-09-28 13:07:15,563: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-09-28 13:07:15,567: INFO: common: yaml file: params.yaml loaded successfully]
[2023-09-28 13:07:15,570: INFO: common: created directory at: artifacts]
[2023-09-28 13:07:15,571: INFO: common: created directory at: artifacts/data_transformation]
[2023-09-28 13:07:17,790: WARNING: arrow_dataset: Loading cached processed dataset at artifacts/data_ingestion/samsum_dataset/train\cache-0cc0f255bad7f608.arrow]
[2023-09-28 13:07:17,824: WARNING: arrow_dataset: Loading cached processed dataset at artifacts/data_ingestion/samsum_dataset/test\cache-e51af59ab391f6b0.arrow]
[2023-09-28 13:07:17,860: WARNING: arrow_dataset: Loading cached processed dataset at artifacts/data_ingestion/samsum_dataset/validation\cache-43419a8b517d5825.arrow]
[2023-09-28 13:07:17,973: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2023-09-28 13:07:17,974: INFO: main: *******************]
[2023-09-28 13:07:17,974: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2023-09-28 13:09:02,978: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-09-28 13:11:12,560: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-10-02 20:14:06,501: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-10-02 20:14:44,587: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-10-02 20:14:57,603: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-10-07 11:50:52,740: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-10-07 11:56:16,322: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-10-07 11:57:12,382: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-12-24 13:20:57,350: INFO: utils: Note: NumExpr detected 16 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.]
[2023-12-24 13:20:57,350: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-12-25 08:36:22,302: INFO: utils: Note: NumExpr detected 16 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.]
[2023-12-25 08:36:22,303: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-12-25 08:53:30,330: INFO: utils: Note: NumExpr detected 16 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.]
[2023-12-25 08:53:30,333: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-12-25 08:53:32,352: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-12-25 08:53:32,362: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-25 08:53:32,364: INFO: common: yaml file: params.yaml loaded successfully]
[2023-12-25 08:53:32,373: INFO: common: created directory at: artifacts]
[2023-12-25 08:53:32,373: INFO: common: created directory at: artifacts/data_ingestion]
[2023-12-25 08:53:32,374: INFO: data_ingestion: File already exists of size: ~ 7718 KB]
[2023-12-25 08:53:32,480: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-12-25 08:53:32,480: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2023-12-25 08:53:32,480: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-25 08:53:32,480: INFO: common: yaml file: params.yaml loaded successfully]
[2023-12-25 08:53:32,480: INFO: common: created directory at: artifacts]
[2023-12-25 08:53:32,480: INFO: common: created directory at: artifacts/data_validation]
[2023-12-25 08:53:32,480: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2023-12-25 08:53:32,480: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2023-12-25 08:53:32,490: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-25 08:53:32,491: INFO: common: yaml file: params.yaml loaded successfully]
[2023-12-25 08:53:32,492: INFO: common: created directory at: artifacts]
[2023-12-25 08:53:32,492: INFO: common: created directory at: artifacts/data_transformation]
[2023-12-25 08:53:37,975: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2023-12-25 08:53:37,976: INFO: main: *******************]
[2023-12-25 08:53:37,976: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2023-12-25 08:53:37,978: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-25 08:53:37,978: INFO: common: yaml file: params.yaml loaded successfully]
[2023-12-25 08:53:37,978: INFO: common: created directory at: artifacts]
[2023-12-25 08:53:37,978: INFO: common: created directory at: artifacts/model_trainer]
[2023-12-25 08:57:33,824: ERROR: main: Using the `Trainer` with `PyTorch` requires `accelerate>=0.20.1`: Please run `pip install transformers[torch]` or `pip install accelerate -U`]
Traceback (most recent call last):
  File "c:\Users\alrik\Git Repositories\Text-Summarization-From-Speech\main.py", line 45, in <module>
    model_trainer.main()
  File "c:\Users\alrik\Git Repositories\Text-Summarization-From-Speech\src\textSummarizer\pipeline\stage_04_model_trainer.py", line 16, in main
    model_trainer_config.train()
  File "c:\Users\alrik\Git Repositories\Text-Summarization-From-Speech\src\textSummarizer\components\model_trainer.py", line 37, in train
    trainer_args = TrainingArguments(
  File "<string>", line 112, in __init__
  File "C:\Users\alrik\anaconda3\envs\Deep_Learning_39\lib\site-packages\transformers\training_args.py", line 1372, in __post_init__
    and (self.device.type != "cuda")
  File "C:\Users\alrik\anaconda3\envs\Deep_Learning_39\lib\site-packages\transformers\training_args.py", line 1795, in device
    return self._setup_devices
  File "C:\Users\alrik\anaconda3\envs\Deep_Learning_39\lib\site-packages\transformers\utils\generic.py", line 54, in __get__
    cached = self.fget(obj)
  File "C:\Users\alrik\anaconda3\envs\Deep_Learning_39\lib\site-packages\transformers\training_args.py", line 1716, in _setup_devices
    raise ImportError(
ImportError: Using the `Trainer` with `PyTorch` requires `accelerate>=0.20.1`: Please run `pip install transformers[torch]` or `pip install accelerate -U`
[2023-12-25 09:11:32,201: INFO: utils: Note: NumExpr detected 16 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.]
[2023-12-25 09:11:32,201: INFO: utils: NumExpr defaulting to 8 threads.]
[2023-12-25 09:11:35,082: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-12-25 09:11:35,082: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-25 09:11:35,082: INFO: common: yaml file: params.yaml loaded successfully]
[2023-12-25 09:11:35,082: INFO: common: created directory at: artifacts]
[2023-12-25 09:11:35,098: INFO: common: created directory at: artifacts/data_ingestion]
[2023-12-25 09:11:35,098: INFO: data_ingestion: File already exists of size: ~ 7718 KB]
[2023-12-25 09:11:35,183: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-12-25 09:11:35,183: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2023-12-25 09:11:35,183: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-25 09:11:35,183: INFO: common: yaml file: params.yaml loaded successfully]
[2023-12-25 09:11:35,183: INFO: common: created directory at: artifacts]
[2023-12-25 09:11:35,183: INFO: common: created directory at: artifacts/data_validation]
[2023-12-25 09:11:35,183: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2023-12-25 09:11:35,183: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2023-12-25 09:11:35,193: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-25 09:11:35,195: INFO: common: yaml file: params.yaml loaded successfully]
[2023-12-25 09:11:35,195: INFO: common: created directory at: artifacts]
[2023-12-25 09:11:35,195: INFO: common: created directory at: artifacts/data_transformation]
[2023-12-25 09:11:36,535: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2023-12-25 09:11:36,535: INFO: main: *******************]
[2023-12-25 09:11:36,535: INFO: main: >>>>>> stage Model Trainer stage started <<<<<<]
[2023-12-25 09:11:36,535: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-25 09:11:36,535: INFO: common: yaml file: params.yaml loaded successfully]
[2023-12-25 09:11:36,535: INFO: common: created directory at: artifacts]
[2023-12-25 09:11:36,535: INFO: common: created directory at: artifacts/model_trainer]
[2023-12-25 15:10:56,753: INFO: main: >>>>>> stage Model Trainer stage completed <<<<<<

x==========x]
[2024-01-02 11:49:09,453: INFO: utils: Note: NumExpr detected 16 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.]
[2024-01-02 11:49:09,453: INFO: utils: NumExpr defaulting to 8 threads.]
[2024-01-02 11:49:15,134: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-01-02 11:49:15,140: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 11:49:15,153: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-02 11:49:15,154: INFO: common: created directory at: artifacts]
[2024-01-02 11:49:15,155: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-02 11:49:15,156: INFO: data_ingestion: File already exists of size: ~ 7718 KB]
[2024-01-02 11:49:15,279: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-01-02 11:49:15,280: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-01-02 11:49:15,282: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 11:49:15,284: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-02 11:49:15,284: INFO: common: created directory at: artifacts]
[2024-01-02 11:49:15,285: INFO: common: created directory at: artifacts/data_validation]
[2024-01-02 11:49:15,287: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-01-02 11:49:15,288: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-01-02 11:49:15,292: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 11:49:15,296: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-02 11:49:15,298: INFO: common: created directory at: artifacts]
[2024-01-02 11:49:15,299: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-02 11:49:17,054: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-01-02 11:49:17,055: INFO: main: *******************]
[2024-01-02 11:49:17,055: INFO: main: >>>>>> stage Model Evaluation stage started <<<<<<]
[2024-01-02 11:49:17,055: ERROR: main: name 'ModelEvaluationTrainingPipeline' is not defined]
Traceback (most recent call last):
  File "c:\Users\alrik\Git Repositories\Text-Summarization-From-Speech\main.py", line 55, in <module>
    model_evaluation = ModelEvaluationTrainingPipeline()
NameError: name 'ModelEvaluationTrainingPipeline' is not defined
[2024-01-02 11:50:26,772: INFO: utils: Note: NumExpr detected 16 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.]
[2024-01-02 11:50:26,772: INFO: utils: NumExpr defaulting to 8 threads.]
[2024-01-02 11:50:29,388: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-01-02 11:50:29,388: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 11:50:29,388: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-02 11:50:29,388: INFO: common: created directory at: artifacts]
[2024-01-02 11:50:29,388: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-02 11:50:29,388: INFO: data_ingestion: File already exists of size: ~ 7718 KB]
[2024-01-02 11:50:29,477: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-01-02 11:50:29,477: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-01-02 11:50:29,477: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 11:50:29,477: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-02 11:50:29,477: INFO: common: created directory at: artifacts]
[2024-01-02 11:50:29,477: INFO: common: created directory at: artifacts/data_validation]
[2024-01-02 11:50:29,487: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-01-02 11:50:29,489: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-01-02 11:50:29,492: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 11:50:29,494: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-02 11:50:29,494: INFO: common: created directory at: artifacts]
[2024-01-02 11:50:29,496: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-02 11:50:30,479: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-01-02 11:50:30,479: INFO: main: *******************]
[2024-01-02 11:50:30,479: INFO: main: >>>>>> stage Model Evaluation stage started <<<<<<]
[2024-01-02 11:50:30,479: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 11:50:30,479: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-02 11:50:30,479: INFO: common: created directory at: artifacts]
[2024-01-02 11:50:30,479: INFO: common: created directory at: artifacts/model_evaluation]
[2024-01-02 11:50:30,492: ERROR: main: Repo id must be in the form 'repo_name' or 'namespace/repo_name': 'artifacts/model_trainer/tokenizer'. Use `repo_type` argument if needed.]
Traceback (most recent call last):
  File "c:\Users\alrik\Git Repositories\Text-Summarization-From-Speech\main.py", line 57, in <module>
    model_evaluation.main()
  File "c:\Users\alrik\Git Repositories\Text-Summarization-From-Speech\src\textSummarizer\pipeline\stage_05_model_evaluation.py", line 16, in main
    model_evaluation_config.evaluate()
  File "c:\Users\alrik\Git Repositories\Text-Summarization-From-Speech\src\textSummarizer\components\model_evaluation.py", line 60, in evaluate
    tokenizer = AutoTokenizer.from_pretrained(self.config.tokenizer_path)
  File "C:\Users\alrik\anaconda3\envs\Deep_Learning_39\lib\site-packages\transformers\models\auto\tokenization_auto.py", line 652, in from_pretrained
    tokenizer_config = get_tokenizer_config(pretrained_model_name_or_path, **kwargs)
  File "C:\Users\alrik\anaconda3\envs\Deep_Learning_39\lib\site-packages\transformers\models\auto\tokenization_auto.py", line 496, in get_tokenizer_config
    resolved_config_file = cached_file(
  File "C:\Users\alrik\anaconda3\envs\Deep_Learning_39\lib\site-packages\transformers\utils\hub.py", line 417, in cached_file
    resolved_file = hf_hub_download(
  File "C:\Users\alrik\anaconda3\envs\Deep_Learning_39\lib\site-packages\huggingface_hub\utils\_validators.py", line 110, in _inner_fn
    validate_repo_id(arg_value)
  File "C:\Users\alrik\anaconda3\envs\Deep_Learning_39\lib\site-packages\huggingface_hub\utils\_validators.py", line 158, in validate_repo_id
    raise HFValidationError(
huggingface_hub.utils._validators.HFValidationError: Repo id must be in the form 'repo_name' or 'namespace/repo_name': 'artifacts/model_trainer/tokenizer'. Use `repo_type` argument if needed.
[2024-01-02 13:23:03,382: INFO: utils: Note: NumExpr detected 16 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.]
[2024-01-02 13:23:03,382: INFO: utils: NumExpr defaulting to 8 threads.]
[2024-01-02 13:23:06,001: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-01-02 13:23:06,028: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 13:23:06,029: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-02 13:23:06,031: INFO: common: created directory at: artifacts]
[2024-01-02 13:23:06,031: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-02 13:23:06,032: INFO: data_ingestion: File already exists of size: ~ 7718 KB]
[2024-01-02 13:23:06,165: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-01-02 13:23:06,165: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-01-02 13:23:06,165: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 13:23:06,165: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-02 13:23:06,165: INFO: common: created directory at: artifacts]
[2024-01-02 13:23:06,165: INFO: common: created directory at: artifacts/data_validation]
[2024-01-02 13:23:06,165: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-01-02 13:23:06,165: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-01-02 13:23:06,179: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 13:23:06,180: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-02 13:23:06,180: INFO: common: created directory at: artifacts]
[2024-01-02 13:23:06,181: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-02 13:23:07,371: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-01-02 13:23:07,373: INFO: main: *******************]
[2024-01-02 13:23:07,373: INFO: main: >>>>>> stage Model Evaluation stage started <<<<<<]
[2024-01-02 13:23:07,376: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 13:23:07,376: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-02 13:23:07,376: INFO: common: created directory at: artifacts]
[2024-01-02 13:23:07,376: INFO: common: created directory at: artifacts/model_evaluation]
[2024-01-02 13:23:07,563: ERROR: main: Can't load the configuration of 'artifacts/model_trainer/pegasus-samsum-model'. If you were trying to load it from 'https://huggingface.co/models', make sure you don't have a local directory with the same name. Otherwise, make sure 'artifacts/model_trainer/pegasus-samsum-model' is the correct path to a directory containing a config.json file]
Traceback (most recent call last):
  File "C:\Users\alrik\anaconda3\envs\Deep_Learning_39\lib\site-packages\transformers\configuration_utils.py", line 672, in _get_config_dict
    resolved_config_file = cached_file(
  File "C:\Users\alrik\anaconda3\envs\Deep_Learning_39\lib\site-packages\transformers\utils\hub.py", line 417, in cached_file
    resolved_file = hf_hub_download(
  File "C:\Users\alrik\anaconda3\envs\Deep_Learning_39\lib\site-packages\huggingface_hub\utils\_validators.py", line 110, in _inner_fn
    validate_repo_id(arg_value)
  File "C:\Users\alrik\anaconda3\envs\Deep_Learning_39\lib\site-packages\huggingface_hub\utils\_validators.py", line 158, in validate_repo_id
    raise HFValidationError(
huggingface_hub.utils._validators.HFValidationError: Repo id must be in the form 'repo_name' or 'namespace/repo_name': 'artifacts/model_trainer/pegasus-samsum-model'. Use `repo_type` argument if needed.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Users\alrik\Git Repositories\Text-Summarization-From-Speech\main.py", line 57, in <module>
    model_evaluation.main()
  File "c:\Users\alrik\Git Repositories\Text-Summarization-From-Speech\src\textSummarizer\pipeline\stage_05_model_evaluation.py", line 16, in main
    model_evaluation_config.evaluate()
  File "c:\Users\alrik\Git Repositories\Text-Summarization-From-Speech\src\textSummarizer\components\model_evaluation.py", line 61, in evaluate
    model_pegasus = AutoModelForSeq2SeqLM.from_pretrained(self.config.model_path).to(device)
  File "C:\Users\alrik\anaconda3\envs\Deep_Learning_39\lib\site-packages\transformers\models\auto\auto_factory.py", line 461, in from_pretrained
    config, kwargs = AutoConfig.from_pretrained(
  File "C:\Users\alrik\anaconda3\envs\Deep_Learning_39\lib\site-packages\transformers\models\auto\configuration_auto.py", line 983, in from_pretrained
    config_dict, unused_kwargs = PretrainedConfig.get_config_dict(pretrained_model_name_or_path, **kwargs)
  File "C:\Users\alrik\anaconda3\envs\Deep_Learning_39\lib\site-packages\transformers\configuration_utils.py", line 617, in get_config_dict
    config_dict, kwargs = cls._get_config_dict(pretrained_model_name_or_path, **kwargs)
  File "C:\Users\alrik\anaconda3\envs\Deep_Learning_39\lib\site-packages\transformers\configuration_utils.py", line 693, in _get_config_dict
    raise EnvironmentError(
OSError: Can't load the configuration of 'artifacts/model_trainer/pegasus-samsum-model'. If you were trying to load it from 'https://huggingface.co/models', make sure you don't have a local directory with the same name. Otherwise, make sure 'artifacts/model_trainer/pegasus-samsum-model' is the correct path to a directory containing a config.json file
[2024-01-02 13:24:27,996: INFO: utils: Note: NumExpr detected 16 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.]
[2024-01-02 13:24:27,996: INFO: utils: NumExpr defaulting to 8 threads.]
[2024-01-02 13:24:30,681: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-01-02 13:24:30,681: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 13:24:30,681: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-02 13:24:30,692: INFO: common: created directory at: artifacts]
[2024-01-02 13:24:30,693: INFO: common: created directory at: artifacts/data_ingestion]
[2024-01-02 13:24:30,693: INFO: data_ingestion: File already exists of size: ~ 7718 KB]
[2024-01-02 13:24:30,793: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-01-02 13:24:30,793: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-01-02 13:24:30,809: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 13:24:30,809: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-02 13:24:30,809: INFO: common: created directory at: artifacts]
[2024-01-02 13:24:30,809: INFO: common: created directory at: artifacts/data_validation]
[2024-01-02 13:24:30,814: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-01-02 13:24:30,815: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-01-02 13:24:30,818: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 13:24:30,819: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-02 13:24:30,820: INFO: common: created directory at: artifacts]
[2024-01-02 13:24:30,821: INFO: common: created directory at: artifacts/data_transformation]
[2024-01-02 13:24:31,828: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-01-02 13:24:31,828: INFO: main: *******************]
[2024-01-02 13:24:31,828: INFO: main: >>>>>> stage Model Evaluation stage started <<<<<<]
[2024-01-02 13:24:31,828: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-01-02 13:24:31,828: INFO: common: yaml file: params.yaml loaded successfully]
[2024-01-02 13:24:31,828: INFO: common: created directory at: artifacts]
[2024-01-02 13:24:31,828: INFO: common: created directory at: artifacts/model_evaluation]
[2024-01-02 13:24:51,402: INFO: rouge_scorer: Using default tokenizer.]
[2024-01-02 13:24:51,497: INFO: main: >>>>>> stage Model Evaluation stage completed <<<<<<

x==========x]
